### PR TITLE
Set up data for new `/teams` page

### PR DESCRIPTION
**Pull request by EthanThatOneKid**

### CODE REVIEW

File: src/routes/(site)/6/teams/members.svelte [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  const members = getMembers(OFFICERS_JSON, data.term, data.team.tiers);
+</script>
+
+<section class="team">
+  <h2>{data.team.title}</h2>
+  <p>{data.team.description}</p>
+  <ul>
+    <!-- TODO: Fix each loop to show the correct officers per term. -->
```
Comment (EthanThatOneKid) [inline src/routes/(site)/6/teams/members.svelte]: TODO(@EthanThatOneKid): Fix bug!

Comment (EthanThatOneKid) [reply src/routes/(site)/6/teams/members.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/955/commits/899530b1092bea858769adea8431667bcc988b31>.

**Review by EthanThatOneKid — COMMENTED** [review]
LGTM!! Looks good on my machine :)

**Review by k4rni — APPROVED** [review]
LGTM!

image

**Pull request merged**